### PR TITLE
Add support for a single project

### PR DESCRIPTION
**Pull request by evilensky**
Filters down to just the specified project.

### CODE REVIEW

File: ssh_generate.py [hunk 1]
```diff
@@ -135,6 +137,10 @@ def main():
         if len(project['followers']) > 0 and org_name == ORG:
             project_names.append(project_name)
 
+    if PROJECT:
+        # filter the list down to just a single project
+        project_names = [ project_name for project_name in project_names if project_name == PROJECT ]
```
Comment (josh-mcq) [inline ssh_generate.py]: how is this any different than:
if PROJECT:
--->project_names = [PROJECT] (my formatting doesn't play nice in pr comments for some reason)

File: ssh_generate.py [hunk 2]
```diff
@@ -135,6 +137,13 @@ def main():
         if len(project['followers']) > 0 and org_name == ORG:
             project_names.append(project_name)
 
+    if PROJECT:
+        if PROJECT in project_names:
+            # filter the list down to just a single project
+            project_names = [ PROJECT ]
+        else:
+            raise Exception("project name {PROJECT} is not found in list of projects".format(PROJECT=PROJECT))
```
Comment (josh-mcq) [inline ssh_generate.py:145]: does this need to be an f string?

Comment (josh-mcq) [reply ssh_generate.py:145]: oh wait, you're using the .format..  that works too.

**Review by josh-mcq — APPROVED** [review]

File: ssh_generate.py [hunk 3]
```diff
@@ -111,6 +111,8 @@ def main():
     # Set to github or bitbucket
     VCS = convert_vcs(sys.argv[3])
 
+    PROJECT = sys.argv[4]
```
Comment (pete-woods) [inline ssh_generate.py:114]: I'm pretty sure this needs to do a bounds check

**Pull request closed without merging**